spec/gitlab_config_spec.rb

```diff
@@ -43,6 +43,12 @@
     it("returns false by default") { should eq(false) }
   end
 
+  describe :audit_client_ip do
+    subject { config.audit_client_ip }
+
+    it("returns false by default") { should eq(false) }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.